model/docs/.gitignore

```diff
@@ -8,3 +8,4 @@ _compiled
 
 *.ipynb
 *.py
+*.rst
```

Comment:
Have we made a decision on the future / necessity of placeholder `.rst` files like [this one](https://github.com/CDCgov/multisignal-epi-inference/blob/main/docs/source/tutorials/basic_renewal_model.rst). @gvegayon can you remind me what the justification for them was?

Reply:
The justification is that "suggest and edit" feature on the docs website points to that file, but the file would not otherwise exist in the repo until it is rendered via the github action.

Reply:
afaict git-tracked `.rst` should not end up in `model/docs` even now (and `model/docs` may no longer exist at all after #233) so this seems fine.